developerportal/apps/articles/templates/articles.html

```diff
@@ -8,6 +8,7 @@
       {% if page.articles %}
         <section>
           <h2>Articles</h2>
+          <p class="page-description">{{ page.description }}</p>
```

Comment:
We should probably have conditional here to only show the `<p>` tag if there's a description, as below.

Reply:
Good point, done.